src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -277,7 +277,11 @@ export default defineConfig({
             // A secret random string that you create.
             bypassToken: "005556d774a8",
             // Paths that will always be served fresh.
-            exclude: [ "/api/invalidate", "/posts/[...slug]" ]
+            exclude: [
+              '/preview', 
+              '/auth/[page]',
+              /^\/api\/.+/ // Regular expressions supported since v8.1.0
```

Comment:
Maybe avoid confusion by saying `@astrojs/vercel@8.1.0`?

Reply:
I mean, it's the page of the Vercel integration, I believe there's no confusion 😅

Reply:
ugh, there is totally going to be confusion. lol I don't love how long that makes it, but we do need to say it.